pkg/metronome/types.go

```diff
@@ -0,0 +1,154 @@
+package metronome
+
+import (
+	"time"
+)
+
+// Job represents a Job returned by the Metronome API.
+type Job struct {
+	ID          string            `json:"id"`
+	Description string            `json:"description"`
+	Labels      map[string]string `json:"labels"`
+	Run         run               `json:"run"`
+
+	// these properties depend on the embed parameters when querying the /v1/jobs endpoints
+	ActiveRuns     []activeRun        `json:"activeRuns,omitempty"`
+	HistorySummary *jobHistorySummary `json:"historySummary,omitempty"`
+	History        *jobHistory        `json:"history,omitempty"`
+	Schedules      []schedule         `json:"schedules,omitempty"`
```

Comment:
I think the nested types in this struct should be exported, they will be needed for other endpoints (`/v1/jobs/{jobId}/runs` or `/v1/jobs/{jobId}/schedules`), it also lets users typehinting them if they want to process them.